src/walletSdk/Anchor/index.ts

```diff
@@ -19,6 +19,7 @@ import { parseToml } from "../Utils";
 type AnchorParams = {
   cfg: Config;
   homeDomain: string;
+  allowHttp?: boolean;
```

Comment:
Wasn't allowHttp replaced with axios client configuration? Or do we still use allowHttp flag for that?
Probably should have it in our documentation 🤔

Reply:
Yup the axios client can use http, but this is for something different. I needed to add because we're using the stellar-sdk Toml Resolver [here](https://github.com/stellar/js-stellar-sdk/blob/0e4a0b02d619dd1ef8c5c51e61c07f730c871581/src/stellartoml/index.ts#L34-L60), which is using their own axios client. And has a `allowHttp` config specific to that

ya good idea I'll add to docs

Reply:
https://github.com/stellar/stellar-docs/pull/305